pkg/extensions/search/common/common.go

```diff
@@ -11,11 +11,13 @@ import (
 )
 
 const (
```

Comment:
https://github.com/opencontainers/image-spec/blob/main/annotations.md
^ why not use these instead?

Reply:
These are the fallbacks. See the table at the end of https://github.com/opencontainers/image-spec/blob/main/annotations.md